synths/core-synths.scd

```diff
@@ -284,6 +284,18 @@ live coding them requires that you have your SuperDirt instance in an environmen
 
 	}, [\ir, \ir]).add;
 
+	// Tremolo audio DSP effect definition. Params: tremolorate (Hz); tremolodepth (amplitude)
+	SynthDef("tremolo" ++ numChannels, { |out, tremolorate = 1.0, tremolodepth = 0.5 |
+		var signal, tremolowave;
+
+		tremolodepth = tremolodepth * 0.5;
+		tremolowave = (1 - tremolodepth) + SinOsc.ar(tremolorate, 0, tremolodepth);
```

Comment:
… ups sorry, ignore the first comment.

you want the tremolowave to range from `1 - tremolodepth` to `1 + tremolodepth`. So whatever! Maybe easiest is: `.range(1 - tremolodepth, 1 + tremolodepth)`.